src/System.Runtime/tests/System/Char.cs

```diff
@@ -748,53 +733,53 @@ public static void TestIsWhiteSpace()
     }
 
     [Fact]
-    public static void TestIsWhiteSpaceStrInt()
+    public static void TestIsWhiteSpace_String_Int()
     {
-        // Boolean Char.IsWhiteSpace(String, Int32)
-        foreach (var c in GetTestChars(UnicodeCategory.SpaceSeparator,
-                UnicodeCategory.LineSeparator,
-                UnicodeCategory.ParagraphSeparator))
+        var categories = new UnicodeCategory[]
+        {
+            UnicodeCategory.SpaceSeparator,
+            UnicodeCategory.LineSeparator,
+            UnicodeCategory.ParagraphSeparator
+        };
+        foreach (char c in GetTestChars(categories))
             Assert.True(char.IsWhiteSpace(c.ToString(), 0));
 
         // Some control chars are also considered whitespace for legacy reasons.
-        //if ((c >= '\x0009' && c <= '\x000d') || c == '\x0085')
-        Assert.True(char.IsWhiteSpace('\u000b'.ToString(), 0));
-        Assert.True(char.IsWhiteSpace('\u0085'.ToString(), 0));
+        // if ((c >= '\x0009' && c <= '\x000d') || c == '\x0085')
+        Assert.True(char.IsWhiteSpace('\u000b'));
+        Assert.True(char.IsWhiteSpace('\u0085'));
```

Comment:
Why did you change these?  You're now using the IsWhiteSpace(char) overload, whereas this test is about IsWhiteSpace(string, int).